src/framework/control.ml

```diff
@@ -154,9 +154,9 @@ struct
 
       (* Adding the state at each system variable to the final result *)
       let add_local_var (n,es) state =
-        let loc = Tracing.getLoc n in
+        let loc = UpdateCil.getLoc n in
```

Comment:
There should be a comment on why this is `UpdateCil.getLoc` here.

Reply:
I added a comment although I'm highly suspicious that there are other places where this needs to be done as well, since this only affects locations in the XML result, not anything else. So some of the other `Node.location` calls which bypass the `UpdateCil` override mapping might still end up in some terminal output.

But given the discussion in #290, it might be a better to change some other things altogether to remove the need for this distinction (and use overrides everywhere).